noxfile.py

```diff
@@ -86,10 +86,7 @@ def default(session):
     session.install(
         "mock", "pytest", "pytest-cov",
     )
-    extras = "fastavro,pandas,pyarrow"
-    if session.python == "3.9":
-        extras = "fastavro,pandas"
-    session.install("-e", f".[{extras}]")
+    session.install("-e", ".[fastavro,pandas,pyarrow]")
```

Comment:
We should probably add these changes to `synth.py` until pyarrow 3.0 is released (https://issues.apache.org/jira/browse/ARROW-11240?jql=project%20%3D%20ARROW%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20fixVersion%20%3D%203.0.0)